helm_chart/ansible-operator/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: ansible-operator
+description: ansible-operator installation chart
```

Comment:
Enrich this. There are several more options that may make your chart more user friendly [ref](https://helm.sh/docs/topics/charts/).